cita-executor/core/src/contracts/solc/version_management.rs

```diff
@@ -30,6 +30,8 @@ use types::ids::BlockId;
 use types::reserved_addresses;
 
 lazy_static! {
+    // Will use `getProtocolVersion` at next version after `v0.25.0`.
+    // And the `getVersion` will be *Deprecated*.
     static ref VERSION_HASH: Vec<u8> = method_tools::encode_to_vec(b"getVersion()");
```

Comment:
Why next version?

Reply:
The new coming version will not enforce user to amend code when upgrade.

Reply:
Ok